assets/templates/aws.ec2_logs/fields.yml

```diff
@@ -0,0 +1,15 @@
+- name: timestamp
+  # TIMESTAMP_ISO8601 %{YEAR}-%{MONTHNUM}-%{MONTHDAY}[T ]%{HOUR}:?%{MINUTE}(?::?%{SECOND})?%{ISO8601_TIMEZONE}?
+  type: date
+- name: syslog_timestamp
+  # SYSLOGTIMESTAMP %{MONTH} +%{MONTHDAY} %{TIME}
+  type: date
+- name: iporhost
+  # This can also be hostname, can we express it?
+  type: ip
+- name: process_name
+  type: keyword
+- name: process_pid
+  type: integer
+- name: message
+  type: text
```

Comment:
@aspacca I checked how to configure a `text` field, but I'm not sure this is currently possible. I'd like to configure length (min/max) for example.

Reply:
there's no such thing at the moment.

the only possibility to "drive" the format of a if you set an `example`, the tool will look for `.-_\s` (in the order) char in the example and create a value that respects the number of "tokens" separated by that char.
for example:
```
- name: InterfaceID
  type: keyword
  example: eni-1235b8ca123456789
```

`{{ generate "InterfaceID" }}` this will generate something like `randomnoun-randomnoun`

we can improve adding more feature

but, most important indeed!, we need to support all the fields type from the specs: https://github.com/elastic/package-spec/blob/main/spec/integration/data_stream/fields/fields.spec.yml#L32-L64

currently we don't: https://github.com/elastic/elastic-integration-corpus-generator-tool/blob/main/pkg/genlib/generator_interface.go#L32-L47

(for example we don't support `text`)